pghoard/walreceiver.py

```diff
@@ -117,6 +121,8 @@ def start_replication(self):
             lsn = LSN(self.last_flushed_lsn, self.pg_version_server)
         else:
             lsn = lsn_from_sysinfo(identify_system, self.pg_version_server)
+        self.initial_lsn = lsn
```

Comment:
I wonder if a property-like implementation which blocks until initial_lsn is set would be better. But this looks good right now; let's think about improving it later.